.github/workflows/gradle.yml

```diff
@@ -37,5 +35,43 @@ jobs:
       run: sudo apt-get install omlib-modelica-3.0
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
-    - name: Test with Gradle
-      run: ./gradlew test -i
+    - name: Build with Gradle
+      run: ./gradlew build -i
```

Comment:
Why `build` and not `test`?

Reply:
This change means we will not get test results and code coverage anymore for each commit.